frontend/src/utils/filterProjectsBySample.js

```diff
@@ -0,0 +1,13 @@
+export default function filterProjectsBySample(setFilter, filters, sampleName, clearFilters) {
```

Comment:
I really don't like these two functions they are really redundant and do next to nothing ... At the very least if you cannot find a more generic solution to the issue try to make this patch more generic ... make one function that is more generic to cover present and future needs. I would suggest  setDefaultFilter(filterName, filterValue, setFilter, filters, clearFilters). This will prevent at least the multiplication of those copy-pasted functions.